Cargo.toml

```diff
@@ -93,21 +93,20 @@ tokio = { version = "1", features = [
   "process",
   "macros",
 ] }
-tokio-serde = "0.8"
+tokio-serde = "0.9"
 tokio-util = { version = "0.7", features = ["codec", "io"] }
 toml = "0.8"
 tower = "0.4"
 url = { version = "2", optional = true }
 uuid = { version = "1.9", features = ["v4"] }
 walkdir = "2"
-# by default which pulls in an outdated failure version
 which = { version = "6", default-features = false }
-zip = { version = "0.6", default-features = false }
+zip = { version = "2", default-features = false }
```

Comment:
Is the problem from #2227 still persist?

If yes, please, try to find the 2.x.y version, that works.

The similar issue on the original repository: https://github.com/zip-rs/zip2/issues/231